### PR TITLE
fix bug stack when dim -1

### PR DESCRIPTION
**Pull request by p517332051**
Signed-off-by: hongwei03 <hongwei03@corp.netease.com>

# Description
When dim is equal to -1, the last dimension is used as the concat dimension by default.But in “stack.cpp”.
So when dim equals -1, assign dim as the last dimension in "stack.cpp".

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
 #931 

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by narendasan** [comment]
@p517332051 Can you add a testcase to verify your fix?

**Comment by narendasan** [comment]
@inocsin Can you review?

**Comment by p517332051** [comment]
> @p517332051 Can you add a testcase to verify your fix?

OK，How can i give you the “testcase”？


First use pytorch export model.
 python code
```
import torch      
torch.manual_seed(0)    
      
DEVICE = torch.device("cuda:0")    
SHAPE = (1,)    
class Model(torch.nn.Module):    
      def __init__(self):    
          super().__init__()    
      
      def forward(self, a):    
          return torch.stack((a, a), -1)    
   
if __name__ == "__main__":    
      tensor = torch.randn(SHAPE, dtype=torch.float32, device=DEVICE)    
      model = Model().eval().to(DEVICE)    
      model = torch.jit.trace(model,tensor)
      out = model(tensor)
      model.save("stack_bug.pt") 
```

Second used C艹 interface test this mode。
C艹 code
```

#include <torch/script.h> // One-stop header.
#include "torch_tensorrt/torch_tensorrt.h"
#include <iostream>
#include <memory>
using namespace  std;
int main(int argc, const char* argv[]) {
    torch::jit::Module module;

    auto mod = torch::jit::load("stack_bug.pt",at::Device(at::DeviceType::CUDA, 0));
    std::cout << "ok\n";
    mod.eval();
    mod.to(at::Device(at::DeviceType::CUDA, 0));
    torch::Tensor in0 = torch::ones({ 1},torch::kFloat).to(at::Device(at::DeviceType::CUDA, 0));
    auto out = mod.forward({in0} );
    out = mod.forward({in0} );
    cout<<"torch_ori: "<<out<<endl;
    //-5.0678 -342.5218 -408.2026
    std::vector<int64_t> InputArraysShape0 = in0.sizes().vec();
    std::vector<std::vector<int64_t> > fixed_sizes = {InputArraysShape0};
    auto compile_settings = torch_tensorrt::ts::CompileSpec(fixed_sizes);
    compile_settings.enabled_precisions = {torch::kFloat};

    compile_settings.truncate_long_and_double = true;
    auto trt_mod = torch_tensorrt::torchscript::compile(mod, compile_settings);
    trt_mod.save("stack_bug_2.pt");
    auto out_trt = trt_mod.forward({in0} );
    cout<<"torch_trt: "<<out_trt<<endl;
    return 0;
}
```

**Comment by p517332051** [comment]
> Labeler / label (pull_request)
I always can not pass “Labeler / label (pull_request)”，Can you teach how to pass it？

**Comment by p517332051** [comment]
@narendasan 
I have add test case in tests.

**Comment by narendasan** [comment]
Don't worry about labeler, it breaks because of some limitations of github.

**Comment by inocsin** [comment]
> @inocsin Can you review?

LGTM

**Review by narendasan — APPROVED** [review]
LGTM
